solidity/contracts/converter/ConverterBase.sol

```diff
@@ -231,26 +213,51 @@ abstract contract ConverterBase is ConverterVersion, IConverter, TokenHolder, Co
     }
 
     /**
-     * @dev withdraws tokens held by the converter and sends them to an account
+     * @dev transfers reserves to a new converter during an upgrade
+     * can only be called by the converter upgraded which should be set at its owner
+     *
+     * @param _newConverter account to receive the new amount
+     */
+    function upgradeReserves(address _newConverter) external override protected ownerOnly {
+        // verify that the converter is inactive or that the owner is the upgrader contract
```

Comment:
comment isn't relevant and we could potentially move the upgrader check to a modifier (not a must).

Reply:
👍